.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
In other repositories, we do use `.eslintrc` file like https://github.com/asyncapi/parser-js/blob/master/.eslintrc.
I would say we can copy from there and remove Typescript related things, among others. 

WDYT?

Reply:
> No, it wasn't. And I think it will be better to remove it from this PR and rather add it later on. The only requirement I would say we should ask for in any PR is to, at least, a basic linting by eslint on each code pushed. Even though running it locally.
_Originally posted by @smoya in https://github.com/asyncapi/spec-json-schemas/pull/452#discussion_r1402729934_

Do we need this here after [your recent comment](https://github.com/asyncapi/spec-json-schemas/pull/452#discussion_r1402729934) about including the JavaScript linting?